### PR TITLE
Build pipeline for Java SDK

### PR DESCRIPTION
**Pull request by kb-kerem**
Adds build pipeline for java sdk

### CODE REVIEW

File: .github/workflows/java-build.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Java (build)
+
+on:
+  push:
+    paths:
+      - 'visual-java/**'
```
Comment (FriggaHel) [inline .github/workflows/java-build.yml:6]:
```suggestion
      - 'visual-java/**'
      - .github/workflows/java-build.yml
```
So the build will be kicked if we only modify the workflow (and won't discover that it fails upon merge)

File: .github/workflows/java-build.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  pull_request:
+    paths:
+      - 'visual-java/**'
```
Comment (FriggaHel) [inline .github/workflows/java-build.yml:9]:
```suggestion
      - 'visual-java/**'
      - .github/workflows/java-build.yml
```

File: .github/workflows/java-build.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '11'
```
Comment (FriggaHel) [inline .github/workflows/java-build.yml:24]: Shouldn't we compile it with Java8 since right now we expect to support starting version 8 ?

Comment (kb-kerem) [reply .github/workflows/java-build.yml:25]: Yes 8 makes more sense, will update

**Review by FriggaHel — APPROVED** [review]

**Pull request merged**